src/index.ts

```diff
@@ -192,14 +198,55 @@ module.exports = class ServerlessPlugin {
       if (config.subscribeToExecutionLogs) {
         await addExecutionLogGroupsAndSubscriptions(this.serverless.service, aws, datadogForwarderArn);
       }
+
+      if (config.subscribeToStepFunctionLogs) {
+        const functionArn = datadogForwarderArn;
```

Comment:
Is there a special reason why we are creating this `functionArn` const just to use once on L236 below?

Reply:
I was running into a type error before when the loop used `forEach` and saving `datadogForwarderArn` to a const resolved the error. It looks like with the `for` loop this is no longer an issue so I removed the `functionArn` const and reference `datadogForwarderArn` directly.